community/kernel/src/main/java/org/neo4j/kernel/impl/api/store/StorePropertyCursor.java

```diff
@@ -78,14 +80,15 @@
     private int offsetAtBeginning;
     private int blocks;
     private int remainingBlocksToRead;
-    private long header;
     private PropertyType type;
     private int keyId;
     private ByteBuffer buffer;
 
     private AbstractDynamicStore.DynamicRecordCursor recordCursor;
     private long currentRecordId;
-    private long[] valuesAfterHeader;
+
+    // in values[0] will be placed the property block header
+    private long[] values = new long[VALUES_SIZE];
```

Comment:
Why not to split those 2 things then? variable for value ( current value[0]?) and all the rest for remaining array?